sdcm/cluster.py

```diff
@@ -3329,7 +3329,7 @@ def __init__(self, targets, params):
         self.local_metrics_addr = start_metrics_server()  # start prometheus metrics server locally and return local ip
         self.sct_ip_port = self.set_local_sct_ip()
         self.grafana_port = 3000
-        self.monitor_branch = self.params.get('monitor_branch', default='branch-2.4')
+        self.monitor_branch = self.params.get('monitor_branch', default='branch-3.0')
```

Comment:
why do we need another default other than the test_default?

Reply:
right Roy, please remove the default value here

Reply:
Done and done